docs/sections/user_guide/cli/tools/config/validate-pass-stdin.out

```diff
@@ -1 +1 @@
-[2024-05-23T19:39:15]     INFO 0 UW schema-validation errors found
+[2024-08-26T22:54:27]     INFO 0 UW schema-validation errors found in config
```

Comment:
This may not seem very valuable, but in the case of drivers that validate both their own config and the `platform:` block, we'll now have e.g.
```
[2024-08-27T00:05:16]     INFO Validating config against internal schema: sfc-climo-gen
[2024-08-27T00:05:16]     INFO 0 UW schema-validation errors found in sfc_climo_gen config
[2024-08-27T00:05:16]     INFO Validating config against internal schema: platform
[2024-08-27T00:05:16]     INFO 0 UW schema-validation errors found in platform config
```
It was probably clear from the order what each message meant, but this is even more explicit.

The specific case for this line involves `uw config validate`, so we have no idea what we're validating except that it's some kind of config.